image/templates/helm/stackrox-central/Chart.yaml.htpl

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2 # Can probably be generalized to v1 later. TODO(ROX-5502).
 name: stackrox-central-services
-icon: https://www.stackrox.com/img/logo.svg
+icon: [< required "" .ChartRepo.IconURL >]
 description: Helm Chart for StackRox Central Clusters
```

Comment:
I just realized that `Helm Chart for StackRox Central Clusters` doesn't seem right? Shouldn't it be `Helm Chart for StackRox Central Service`? A quick search in our repo it seems like we call it `Central Service` way more often than `Central Clusters`.

Reply:
Well spotted! Fixed :)